src/NuGet.Core/NuGet.Protocol/Plugins/PluginFile.cs

```diff
@@ -21,19 +21,19 @@ public sealed class PluginFile
         public Lazy<PluginFileState> State { get; }
 
         /// <summary>
-        /// Is the plugin file, a dotnet tools plugin file?
+        /// Indicates if the plugin file is runnable, such as an executable or a script.
         /// </summary>
-        internal bool IsDotnetToolsPlugin { get; }
+        internal bool IsRunnablePluginFile { get; }
```

Comment:
Was there a reason we wanted to move away from IsDotnetToolsPlugin? I was going to suggest `RequiresDotnetHost` but I'm not sure if we're trying to avoid including "dotnet" in the name.

Reply:
the plugin can be written in any languaege, it doesn't have to be a .NET tool